src/Elmish.WPF/Merge.fs

```diff
@@ -66,10 +66,6 @@ module CollectionTarget =
       Enumerate = ct.Enumerate
       GetCollection = ct.GetCollection >> fOut }
 
-  let map outMapA outMapCollection inMapA =
-    mapA outMapA inMapA
-    >> mapCollection outMapCollection
-
 
```

Comment:
When we add the `'t` type parameter to `BindingData`, we are promoting types like `'aCollection` from a "minor" type to a major type. Subsequently (to maintain backwards compatibility) `Helpers.createBinding` boxes one of the *major* types of `BindingData` (ie, `'t`) so that we don't break existing code.

In one of the next PRs, we will be adding something like `Helpers.createBindingT` that doesn't box the major type, and then use it in the new `StaticHelper` class.

Reply:
Pin this for a later PR.